extensions/resteasy-classic/rest-client-config/runtime/src/main/java/io/quarkus/restclient/config/RestClientBuildConfig.java

```diff
@@ -0,0 +1,18 @@
+package io.quarkus.restclient.config;
+
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class RestClientBuildConfig {
+
+    /**
+     * The CDI scope to use for injection. This property can contain either a fully qualified class name of a CDI scope
+     * annotation (such as "jakarta.enterprise.context.ApplicationScoped") or its simple name (such as
+     * "ApplicationScoped").
```

Comment:
I would indicate what's the default if not set.

Reply:
Done